octobot_commons/evaluators_util.py

```diff
@@ -22,7 +22,8 @@
 UNSET_EVAL_TYPE = "unset_eval_type_param"
 
 
-def check_valid_eval_note(eval_note, eval_type=UNSET_EVAL_TYPE, expected_eval_type=None):
+def check_valid_eval_note(eval_note, eval_type=UNSET_EVAL_TYPE, expected_eval_type=None,
```

Comment:
Can you update the cython header please ?

Reply:
Right, surpisingly no CI failed

Reply:
:thinking: